README.md

```diff
@@ -11,11 +11,11 @@ This branch contains Gitlab e2e pipeline for litmus-portal with BDD tests using
   </tr>
   <tr>
     <td>Setup</td>
-    <td>This stage is used for connecting to the OnPrem cluster which is a 4 node cluster(1 master and 3 worker) and clone the litmus-e2e folder.</td>
+    <td>This stage is used for creating a 3-node cluster on EKS.</td>
```

Comment:
Instead of mentioning Stages, we can follow this formate here https://github.com/litmuschaos/litmus-e2e/blob/generic/README.md

Reply:
We can update this later as well.